drkafka/pom.xml

```diff
@@ -82,14 +82,8 @@
 		</dependency>
 		<dependency>
```

Comment:
add comments about the issues that we encountered with other versions of junit-jupiter packages?